src/commands/session.ts

```
@@ -159,3 +198,258 @@ export const sessionCommand = define({
+	const allEntries: Array<{
+		data: UsageData;
+		timestamp: string;
+		cost: number;
+		sessionKey: string;
+		model: string | undefined;
+	}> = [];
+
+	// Load raw data to get all entries
+	// This is a temporary solution - ideally we'd refactor loadSessionData to return raw entries
+	const claudePath = getDefaultClaudePath();
+	const path = await import('node:path');
```

ESLint

[error] Unsafe assignment of an `any` value.

(ts/no-unsafe-assignment)